### PR TITLE
Updates to venues

### PR DESCRIPTION
**Pull request by breethomas**
@robbiejaeger You're suggestions were almost right. Please review. And merge at your discretion.

### CODE REVIEW

File: venue.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Venue List
 
-  * Chuckie Cheese
+  * Chuckie Cheese (VETO)
```
Comment (robbiejaeger) [inline venue.md]: Fine...I agree.

File: venue.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
   * Lucky Strike
   * The Bahamas
+
+  Let's consider...
+
+  * Spain
+  * France
```
Comment (robbiejaeger) [inline venue.md]: Too expensive. Can we take this off the list? @breethomas

File: venue.md (hunk 1 above)
Comment (breethomas) [inline venue.md]: 😺

File: venue.md (hunk 2 above)
Comment (breethomas) [inline venue.md]: No.

**Pull request merged**